src/java/org/apache/cassandra/auth/CassandraRoleManager.java

```diff
@@ -110,8 +98,24 @@ public class CassandraRoleManager implements IRoleManager, CassandraRoleManagerM
     public static final String DEFAULT_SUPERUSER_NAME = "cassandra";
     public static final String DEFAULT_SUPERUSER_PASSWORD = "cassandra";
 
+    /**
+     * Role options which are supported for all authentication mechanisms. IAuthenticator implementations can declare
+     * additional supported role options via {@link IAuthenticator#getSupportedRoleOptions()}.
+     */
+    @VisibleForTesting
+    static final Set<Option> DEFAULT_SUPPORTED_ROLE_OPTIONS = ImmutableSet.of(Option.LOGIN, Option.SUPERUSER);
```

Comment:
what's wrong with `Set.of(Option.LOGIN, Option.SUPERUSER) / Set.of()`, below as well? That is immutable as well.